### ISSUE TITLE
api docs: parent classes in signatures are distracting, can they only show when expanded?

### ISSUE THREAD

**Issue by bishabosha** [lampepfl/dotty#16126]
<img width="803" alt="Screenshot 2022-09-30 at 11 07 49" src="https://user-images.githubusercontent.com/13436592/193236016-7368cb25-d169-492d-b081-0631747dbb91.png">

also I think the names should be aligned, with "modifiers" expanding to the left, like scala 2 docs, otherwise its harder to scan the page

**Comment by odersky** [comment]
I agree. Furthermore, they really should use `,` instead of `with`. `,` is idiomatic for Scala 3 and will help with layout. 

Besides that, I find the Scala 2 docs **hugely** more readable than the Scala 3 docs. In particular is you need to find something which is the standard (maybe the only?) mode we interact with docs. I get drowned in a sea of whitespace and don't find anything. This might be fashionable for essays and such but if you need to look up anything it's a really bad UX. 

Is there a way to go back to the Scala 2 layout rules? It's such an unnecessary distraction that the Scala 3 docs suck so much.

**Comment by szymon-rd** [comment]
Right now we are doing a big rework of the UI. The API members list is one of the things that we are changing. Here you can see the new design of members list:
<img width="1267" alt="image" src="https://user-images.githubusercontent.com/4761866/193258493-b636b414-e1fe-4734-b143-cddbc87bae47.png">
I think that way it's easier to work with when you want to look something up. Right now it is being implemented, along with a lot of changes including:
 - Reworked search engine (faster & gives the expected results)
 - A lot of UX/UI changes like font rendering, margins and paddings
 - ToC in APIs
 - Better filters on API members
 - The page will be loading faster and won't flash on load (likely, that one is a bit problematic because requires a rework of the way the page is loaded, but we are getting there to fix all bugs caused by it)
 - Scroll bars.
 - A lot of bug fixes & other issues.
 
 We will be creating a PR as soon as we get the new members UI to work.
 Here is the branch we are working on:
 https://github.com/szymon-rd/dotty/tree/scaladoc-fixes

**Comment by Dedelweiss** [comment]
Hello, can I help for this issue ? It seems to me that the main problem is solved but the Issue is not closed.
